table8.do

```diff
@@ -1,174 +1,168 @@
-/*regressions adjusted city names new and new race categories */
-/*------------------------------------------------------------*/	
-
-foreach yvar in $YVARS {
-	foreach cluster in $CLUSTER {
-		foreach dvar in $DVAR {
-		
-		qui reghdfe `yvar' i.`dvar' `yvar'_ad ${CONTVARS}, absorb(${ABSVARSSAME} temp_city) keepsingle cluster(`cluster')
-		qui eststo `yvar'_`dvar'_`cluster'_ca
-		qui estadd local ln_price "Yes", replace
-		qui estadd local race_compo "Yes", replace
-		qui estadd local ad_home "Yes", replace
-		}
-	}
-}
-
-
-
-
-/*------------------------------------------------------------*/
-/*Generating Outputs */
-/*------------------------------------------------------------*/	
-
-label variable oc "Racial Minority"
-label variable noc "Racial Minority"
-
-global YVAR "ranking pov"
-
-
-foreach yvar in $YVAR {
-	foreach cluster in $CLUSTER {
-	
-	esttab `yvar'_oc_`cluster' `yvar'_noc_`cluster' `yvar'_noc_`cluster'_ca using "${OUTPUT}/row1_`cluster'_`yvar'.tex" ///
-	, b(%8.3f) se(%8.3f) ///
-	replace booktabs label ///
-		mgroups("Original Data" "Correct Race Only" "Updated City Name and Correct Race",pattern(1 1 1) ///
-		prefix(\multicolumn{@span}{c}{) suffix(}) span erepeat(\cmidrule(lr){@span})) ///
-		title(School Quality and Neighbourhood Safety: Housing Search Platform (Elementary School) (Panel A)) ///
-		alignment(c) page(dcolumn) nomtitle ///
-		cells("b(star fmt(4))" se ci(fmt(4) par)) ///
-		starlevels(* 0.10 ** 0.05 *** 0.01) ///
-		s(ln_price race_compo ad_home N r2_a num_cities, ///
-		label("ln(price) advertised home" "Racial composition, advertised home" "Outcome, advertised home" "Observations" "Adjusted R$^2$" "Number of Cities")) ///
-		keep(`racial_minority')
-		
-	}
-		
-}
-
-foreach yvar in $YVAR {
-	foreach cluster in $CLUSTER {
-	
-	esttab `yvar'_race_`cluster' `yvar'_nrace_`cluster' `yvar'_nrace_`cluster'_ca  using "${OUTPUT}/row2_`cluster'_`yvar'.tex" ///
-	, b(%8.3f) se(%8.3f) ///
-	replace booktabs label ///
-		mgroups("Original Data" "Correct Race Only" "Updated City Name and Correct Race",pattern(1 1 1) ///
-		prefix(\multicolumn{@span}{c}{) suffix(}) span erepeat(\cmidrule(lr){@span})) ///
-		title(American Community Survey: Poverty Rate (Panel B)) ///
-		alignment(c) page(dcolumn) nomtitle ///
-		cells("b(star fmt(4))" se ci(fmt(4) par)) ///
-		starlevels(* 0.10 ** 0.05 *** 0.01) ///
-		s(ln_price race_compo ad_home N r2_a num_cities, ///
-		label("ln(price) advertised home" "Racial composition, advertised home" "Outcome, advertised home" "Observations" "Adjusted R$^2$" "Number of Cities")) ///
-		
-	}
-		
-}
-
-
-// Manual City counter
-
-	//For number of hcityx
-	
-		generate id = _n
-		sort hcityx order
-		by x: gen count = _n == 1
-		
-		
-	// For number of tempcity
-	
-		generate id = _n
-		sort temp_city order
+/* Stata Do File for Table 8 */
+/* Written by: Matthew D. Webb */
+/* Updated: July 30th, 2024, by Sunny Karim */
+
+clear
+
+import delimited "${Data}/Table 8.csv", case(preserve)
```

Comment:
This csv file is not available from the 'Data' folder. Suggest to remove this line and let line 19 call the dta file instead.